bundles/net.tourbook/src/de/byteholder/geoclipse/mapprovider/LayerOfflineData.java

```diff
@@ -1,42 +1,44 @@
 /*******************************************************************************
- * Copyright (C) 2005, 2009  Wolfgang Schramm and Contributors
- *   
+ * Copyright (C) 2005, 2020  Wolfgang Schramm and Contributors
+ *
  * This program is free software; you can redistribute it and/or modify it under
- * the terms of the GNU General Public License as published by the Free Software 
+ * the terms of the GNU General Public License as published by the Free Software
  * Foundation version 2 of the License.
- *  
- * This program is distributed in the hope that it will be useful, but WITHOUT 
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
  * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
- * 
- * You should have received a copy of the GNU General Public License along with 
+ *
+ * You should have received a copy of the GNU General Public License along with
  * this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA    
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
  *******************************************************************************/
 package de.byteholder.geoclipse.mapprovider;
 
+import net.tourbook.common.UI;
+
 public class LayerOfflineData implements Cloneable {
 
-	public String	name;
-	public String	title;
+   public String  name;
+   public String  title;
 
-	public boolean	isDisplayedInMap;
-	public int		position;
+   public boolean isDisplayedInMap;
+   public int     position;
 
-	@Override
-	protected Object clone() throws CloneNotSupportedException {
+   @Override
+   protected Object clone() throws CloneNotSupportedException {
 
-		final LayerOfflineData offlineData = (LayerOfflineData) super.clone();
+      final LayerOfflineData offlineData = (LayerOfflineData) super.clone();
 
-		offlineData.name = name == null ? null : new String(name);
-		offlineData.title = title == null ? null : new String(title);
+      offlineData.name = name == null ? null : new String(name);
+      offlineData.title = title == null ? null : new String(title);
 
-		return offlineData;
-	}
+      return offlineData;
+   }
 
-	@Override
-	public String toString() {
- 		return "p:" + position + " v:" + isDisplayedInMap + " " + title + "(" + name + ")"; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$ //$NON-NLS-4$ //$NON-NLS-5$
-	}
+   @Override
+   public String toString() {
+      return "p:" + position + " v:" + isDisplayedInMap + UI.SPACE1 + title + UI.SYMBOL_BRACKET_LEFT + name + UI.SYMBOL_BRACKET_RIGHT; //$NON-NLS-1$ //$NON-NLS-2$
+   }

```

Comment:
All `toString()` methods are mainly used for debugging and often changed during development, so converting static texts to constants should be skiped for these methods.

Reply:
Fixed